tremor-cli/src/main.rs

```diff
@@ -126,6 +123,10 @@ async fn run(cli: Cli) -> Result<()> {
         Command::Run(r) => r.run().await,
         Command::Doc(d) => d.run(),
         Command::New { name } => create_template(std::env::current_dir()?, &name),
+        Command::PluggableLogging {} => {
```

Comment:
We can remove this command, as pluggable logging is not a stand-alone thing, but works with the `--pluggable-logging` option for all other commands

Reply:
This seems to be gone, can we resolve this?